ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -265,7 +264,12 @@ class GraphsContainer extends React.Component {
           }`}
           ref={this.tooltip}
         >
-          <span className="close mr-3 my-2 ml-2" onClick={this.closeTooltip}>
+          <span
```

Comment:
Same as my comment above it is probably better to use a Button element where possible.